app/services/property_search.rb

```diff
@@ -16,16 +16,12 @@ def perform
   def id_query
     {
       property: property,
-      cautionaryAlerts: cautionary_alerts
+      alert: cautionary_alerts
```

Comment:
I think this might worth named as `alerts` since it will be a hash of `locationAlerts` and `personAlerts`.